### PR TITLE
Updating SLO module and amending codeowners

### PR DESCRIPTION
**Pull request by chrisdodd93**
# Description:
Updating SLO dashboard module, amending codeowners and updating service health dashboard

## Checklist:
- [ ] Is my change backwards compatible? Please include evidence
- [ ] I have tested this and added output to Jira Comment:
- [ ] Documentation added (link) Comment:

### CODE REVIEW

File: dashboards.tf [hunk 1]
```diff
@@ -20,26 +20,10 @@ resource "dynatrace_dashboard_sharing" "signin-signup" {
...
-    permission {
-      id    = data.dynatrace_iam_group.all.id
-      level = "VIEW"
-      type  = "GROUP"
-    }
-  }
+module "slo_dashboard" {
+  count  = local.is_production ? 1 : 0
+  source = "./modules/dashboard"
+  path   = "slos.json"
```
Comment (rjbaker) [inline dashboards.tf:26]: Assuming this resolves to the correct file path

**Review by rjbaker — APPROVED** [review]

File: dashboards.tf (hunk 1 above)
Comment (s18n) [reply dashboards.tf:26]: https://github.com/govuk-one-login/observability-configuration/blob/096eb1dc68d35868bcd7c603f2efea75507bedf1/modules/dashboard/main.tf#L2 

checks out, as long as terraform is executed in the root of this project it should pickup the right file

Comment (chrisdodd93) [reply dashboards.tf:26]: It's all automated via the GitHub actions, but yes you're right Sam, this particular dashboard is in the root of the dashboards folder and hence why the "path" is only the name of the dashboard json file.

**Comment by chrisdodd93** [comment]
I'm going to close this PR and encompass the changes into a wider PR to include additional dashboard changes requested by TSD.

**Pull request closed without merging**